src/lava/proc/lif/models.py

```diff
@@ -7,29 +7,131 @@
 from lava.magma.core.model.py.ports import PyInPort, PyOutPort
 from lava.magma.core.model.py.type import LavaPyType
 from lava.magma.core.resources import CPU
-from lava.magma.core.decorator import implements, requires
+from lava.magma.core.decorator import implements, requires, tag
 from lava.magma.core.model.py.model import PyLoihiProcessModel
 from lava.proc.lif.process import LIF
 
 
 @implements(proc=LIF, protocol=LoihiProtocol)
 @requires(CPU)
-class PyLifModel(PyLoihiProcessModel):
+@tag('floating_pt')
+class PyLifModelFloat(PyLoihiProcessModel):
```

Comment:
Could you write a docstring here to explain what the point of this ProcessModel is (default Process Model based on Python and using floating point numbers)?

Reply:
Done.